app/src/main/java/vn/edu/usth/minigh/SplashActivity.java

```diff
@@ -0,0 +1,38 @@
+package vn.edu.usth.minigh;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+
+import java.util.Timer;
+import java.util.TimerTask;
+
+public class SplashActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_splash);
+    }
```

Comment:
I'd prefer overriding the constructor for the ease of inheritance but in this case it does not matter.

Reply:
LOL we have that when we create the activity in Android Studio, let's skip this.